### PR TITLE
[Cognitive Services] March release (add usages operation, addnew/decomission some APIs)

### PR DESCRIPTION
**Pull request by felixwa**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json [hunk 1]
```diff
@@ -622,24 +682,21 @@
       "type": "string",
       "description": "Required. Indicates the type of cognitive service account.",
       "enum": [
-        "Academic",
-        "Bing.Autosuggest",
```
Comment (felixwa) [inline specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json]: Academic - decommissioned from Ibiza to Labs.
Bing.Autosuggest/Search/SpellCheck - replaced by their .v7 version.
Recommendations - decommissioned.
Speech - decommissioned, replaced by Bing.Speech.

**Comment by sarangan12** [comment]
@felixwa Thanks for specifying the reason for removals in enum. But, I am wondering if the removals are happening in the same version or are you removing it in new version? Because, removing it in the existing versions will be a breaking change to the SDK users. Are you ok with that?

**Comment by felixwa** [comment]
@sarangan12 , for version do you mean RP version? (2017-04-18). If so yes, we would like to remove it in this version as these APIs are either decommissioned or replaced by new versions. Note this field has model-as-string = true, so only constant strings will be disappeared from SDK, if user uses original string he/she can still get the account (but creation is disabled as they have been decommissioned).

**Comment by felixwa** [comment]
APIs in Cognitive Services are quite dynamic, meaning from time to time there will be new APIs added while some decommissioned.  Let me know if there is better way to express them in Swagger.

**Comment by felixwa** [comment]
@sarangan12 , do you have follow up comment on this? Otherwise can you approve the PR?

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**